Dev

TechnologicNick opened the pull request:
Bugfixes and changed README.md

Xesau reviewed (commented):
toppie

djsime1 commented:
Racing drivers seat errors in-game, visual skeleton bugs (primarily with the hands)

djsime1 reviewed (changes requested):
Racing drivers seat errors in-game, visual skeleton bugs (primarily with the hands)

TechnologicNick commented:
# Changelog
Added GUI to Orientation Block

Reworked Part Spawner visuals

Fixed animation errors of Keypad
Fixed animation errors of WASD Thruster
Fixed animation errors of Gimbal Thruster
Fixed animation errors of Keypad
Fixed animation errors of Large Wing
Fixed animation errors of Small Wing

Changed type to scripted of Motorcycle Seat
Changed type to scripted of Racing Diver Seat
Changed type to scripted of Ship's Wheel Seat
Changed type to scripted of Jetpack Seat
Changed type to scripted of Standing Driver Seat
Changed type to scripted of Modular Seat 1 - Standard
Changed type to scripted of Modular Seat 2 - Saddle
Changed type to scripted of Modular Seat 3 - Race
